### ISSUE TITLE
[事前学習] - Scaling Law for Data Mixture in Multilingual Pre-training

### ISSUE THREAD

**Issue by XuyangLLM** [llm-jp/pretrain-tasks#257]
# Overview

We are constructing a scaling law for data mixture in multilingual pre-training. By leveraging it, it is possible to predict the performance of different combinations of language train corpora with ratios. It will benefit in solving English source far exceeds that of other languages in multilingual pre-training.

# Details

モデルカードPR: https://github.com/llm-jp/model-cards/pull/{id}

We are going to pre-train 1.8B or 3.7B models using NII's parameters and settings in order to observe how the final test loss change and fit with our proposed scaling law. The experiments mainly include over 63 combinations, and 7 ratios of six languages for constructing train corpora used for pre-training. 

# Resources

* **計算機**
  * クラスタ: **FIXME** ABCI
  * ノード種別: **FIXME** gpu (H200x8)
  * ノード台数: **FIXME** 1
* **コード**
  * リポジトリ: **FIXME** https://github.com/{org}/{repo}
  * コミット: **FIXME** xxxxxx
* **入力データ**:
  * {name}: `{physical path}`
* **出力データ**:
  * 保存先: `{cluster}:/data/experiments/{number}`
  * データ内訳:
    * {name}: xxx TB （バッファ容量を含む）
* **W&B ログ**:
  * https://wandb.ai/{team}/{project} **FIXME**
* **開始日**: 2025-10-29
* **終了予定日**: 2025-12-31（バッファ期間を含む）
